packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/utils/getDisplayedFilters.js

```diff
@@ -13,6 +13,26 @@ const customOperators = [
 ];
 
 const getDisplayedFilters = ({ formatMessage, users }) => {
+  const getDisplaynameFromUser = (user) => {
```

Comment:
It would be great if you could start adding tests for the whole util (can be done as a follow-up PR).